app/scripts/proactive/model/Job.js

```diff
@@ -120,6 +121,7 @@ define(
           },
           itemToString: Utils.inlineNameValue,
           itemTemplate: bigCrossTemplate,
+          confirmDelete: 'You are about to remove a property.',
```

Comment:
Should not be a question? something like "are you really really sure you want to remove this property? :("